pom.xml

```diff
@@ -17,7 +17,7 @@
     the version "version.xyz.block.web5" set this web5-parent. When updating Web5, update both
     this version *and* the <version.xyz.block.web5> below in the <properties> section.
     -->
-    <version>1.0.0</version>
+    <version>1.1.0</version>
   </parent>
```

Comment:
I don't think we need to update this version number ourselves? @ALRubinger

Reply:
Yes, if we're upgrading `tbdex-kt` to `web5-kt:1.1.0`, updating the version number in both places as this PR does is the right way.

Are the comments in the POM not describing that clearly enough? If not, let's update the comments!